src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -90,37 +97,36 @@ import scala.runtime.Statics
   *  // 5
   *  }}}
   *
-  *  There are a number of subtle points to the above example.
+  *  There are some subtleties here.
   *
-  *  - The definition of `fibs` is a `val` not a method.  The memoization of the
+  *  - The definition of `fibs` uses `val` not `def`.  The memoization of the
   *  `LazyList` requires us to have somewhere to store the information and a `val`
   *  allows us to do that.
   *
-  *  - While the `LazyList` is actually being modified during access, this does not
-  *  change the notion of its immutability.  Once the values are memoized they do
-  *  not change and values that have yet to be memoized still "exist", they
-  *  simply haven't been realized yet.
+  *  - Though the `LazyList` changes as it is accessed, this does not
+  *  contradict its immutability.  Once the values are memoized they do
+  *  not change. Values that have yet to be memoized still "exist", they
+  *  simply haven't been computed yet.
```

Comment:
is it worth mentioning that it is impossible to "access an unmemoized element", so to speak?